src/HeckeMiscMatrix.jl

```diff
@@ -712,3 +712,98 @@ function map_entries(R::ZZModRing, M::ZZMatrix)
     end
     return N
 end
+
+################################################################################
+#
+#  Eigenvalues and eigenspaces
+#
+################################################################################
+
+@doc raw"""
+    eigvals(M::MatElem{T}) where T <: FieldElem
```

Comment:
```suggestion
    eigenvalues(M::MatElem{T}) where T <: FieldElem
```
use the new name everywhere